src/request/src/response.js

```diff
@@ -150,18 +183,20 @@ export class KinveyResponse extends Response {
         || name === 'BlobNotFound'
         || name === 'DocumentNotFound') {
       error = new NotFoundError(message, debug, code, kinveyRequestId);
-    } else if (name === 'ParameterValueOutOfRangeError') {
+    } else if (name === 'ParameterValueOutOfRange') {
       error = new ParameterValueOutOfRangeError(message, debug, code, kinveyRequestId);
     } else if (name === 'ServerError') {
       error = new ServerError(message, debug, code, kinveyRequestId);
+    } else if (name === 'StaleRequest') {
+      error = new StaleRequestError(message, debug, code, kinveyRequestId);
+    } else if (name === 'UserAlreadyExists') {
+      error = new UserAlreadyExistsError(message, debug, code, kinveyRequestId);
+    } else if (name === 'WritesToCollectionDisallowed') {
+      error = new WritesToCollectionDisallowedError(message, debug, code, kinveyRequestId);
     } else {
       return super.error;
```

Comment:
@thomasconner what is `super.error` here? If `name` is not one of the errors we expect, what will happen?